website/docs/r/secret.html.markdown

```diff
@@ -21,15 +21,18 @@ resource "secrethub_secret" "ssh_key" {
 }
 ```
 
-To generate a new secret:
+To generate a new secret made of lowercase letters and symbols, with minimum 5 symbols:
 
 ```terraform
 resource "secrethub_secret" "db_password" {
   path = "company/repo/db_password"
 
   generate {
-    length      = 20
-    use_symbols = true
+    length   = 20
+    charsets = ["lowercase", "symbols"]
+    min      = {
+        symbols = 5
```

Comment:
Maybe show the simplest usage first, then show more complex possibilities?

Reply:
Good idea. I'll add another, more simple example.